.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing
+
+Please read EasyPost's general [contribution guidelines](https://github.com/EasyPost/.github/blob/main/CONTRIBUTING.md)
```

Comment:
are these top two statements sentences? should we end them with a `.`? i'm not a grammar expert lol so i'm curious

Reply:
good catch, i think all the sentences here, even the simple sentences should end with a period.